phpunit/class-block-library-navigation-link-test.php

```diff
@@ -212,6 +212,35 @@ public function test_returns_link_for_plain_link() {
 		);
 	}
 
+	public function test_returns_link_for_decoded_link() {
+		$parsed_blocks = parse_blocks(
+			'<!-- wp:navigation-link {"label":"My Website","url": "https://example.com/?data=lzB%252Fzd%252FZA%253D%253D"} /-->'
+		);
+		$this->assertEquals( 1, count( $parsed_blocks ) );
+
+		$navigation_link_block = new WP_Block( $parsed_blocks[0], array() );
+
+		echo(
+				render_block_core_navigation_link(
+					$navigation_link_block->attributes,
+					array(),
+					$navigation_link_block
+				)
+		);
+
+		$this->assertEquals(
+			true,
+			strpos(
+				render_block_core_navigation_link(
```

Comment:
```suggestion
				gutenberg_render_block_core_navigation_link(
```

Gutenberg apparently has a build system that adds a `gutenberg_` prefix to block functions. You'll want to add `gutenberg_` prefix to your function call in order to execute the plugin version of the function.

I submitted a PR to fix these in the other tests: https://github.com/WordPress/gutenberg/pull/46662